.github/workflows/repo-plan.toml

```diff
@@ -0,0 +1,14 @@
+[current-repo]
+name = "fuel-core"
+owner = "FuelLabs"
+
+[repo.fuels-rs.details]
+name = "fuels-rs"
+owner = "FuelLabs"
+
+[repo.fuel-core.details]
+name = "fuel-core"
+owner = "FuelLabs"
+
+[repo.fuels-rs]
+dependencies = ["fuel-core"]
```

Comment:
Yeah two files are not the same because their current repo name's are different (one of them is `fuels-rs` other is `fuel-core`). I will iterate over this and move repo-plan to releasy's repo and host it in a single location, and fetch repo name etc from the action later on but for now all repos needs to have the repo-plan described to them.